docs/build-for-developers/cli-usage.md

```diff
@@ -1,98 +1,137 @@
+| --------------------------------------------- | -------------------------------------------------------- |
+| `openfn path/to/job.js -a adaptor -l none`    | Quiet mode                                               |
+| `openfn path/to/job.js -a adaptor -l default` | Top level information of what is happening               |
+| `openfn path/to/job.js -a adaptor -l info`    | Get more feedback about runtime, cli and the job         |
+| `openfn path/to/job.js -a adaptor -l debug`   | Get information about runtime, cli, compiler and the job |
+
+### Maintain auto-installed adaptors repo
+
+**List the repo contents**
 
 ```bash
-openfn hello.js -a http -l none
+openfn repo list
 ```
 
-| log level    | description                                              |
-| ------------ | -------------------------------------------------------- |
-| `-l none`    | Quiet mode                                               |
-| `-l default` | Top level information of what is happening               |
-| `-l info`    | Get more feedback on what is happening openfn            |
-| `-l debug`   | Get information about runtime, cli, compiler and the job |
+**Specify the repo folder using the `OPENFN_REPO_DIR` env var**
 
-### Compilation
+```bash
+export OPENFN_REPO_DIR=/path/to/repo
+```
+
+**Auto-install adaptors and check if a matching version is found in the repo**
+
+```bash
+openfn path/to/job.js -ia adaptor-name
+```
 
-The CLI will attempt to compile your job code into normalized Javascript. It
-will do a number of things to make your code robust, portable, and easier to
-debug from a pure JS perspective.
+**Remove all adaptors from the repo**
 
 ```bash
-openfn compile [path]
+openfn repo clean
 ```
 
-Will compile the openfn job and print or save the resulting js.
+### Run a workflow
+
+<details>
+  <summary>A workflow has a structure like this</summary>
+
+```json
+{
+  "start": "a", // optionally specify the start node (defaults to jobs[0])
+  "jobs": [
+    {
+      "id": "a",
+      "expression": "fn((state) => state)", // code or a path
+      "adaptor": "@openfn/language-common@1.75", // specifiy the adaptor to use (version optional)
+      "data": {}, // optionally pre-populate the data object (this will be overriden by keys in in previous state)
+      "configuration": {}, // Use this to pass credentials
+      "next": {
+        // This object defines which jobs to call next
+        // All edges returning true will run
+        // If there are no next edges, the workflow will end
+        "b": true,
+        "c": {
+          "condition": "!state.error" // Note that this is an expression, not a function
+        }
+      }
+    }
+  ]
+}
+```
+
+</details>
+
+To run a workflow
+
+```bash
+openfn path/to/workflow.json -i
+```
 
-Learn more about CLI
-[github.com/OpenFn/kit/](https://github.com/OpenFn/kit/tree/main/packages/cli)
+Here is the detailed tutorial about
+[running workflows](cli-tutorial#8-running-workflows)
```

Comment:
Replace this sentence with the following "Check out this detailed [tutorial](cli-tutorial#8-running-workflows) on running workflows via the CLI"